### PR TITLE
Create accessibility-guidelines

### PR DESCRIPTION
**Pull request by kristenjernigan**
Create start of accessibility guidelines page for project management practice area

----
📚 Documentation preview 📚: https://civicactions-handbook--1327.org.readthedocs.build/en/1327/

### CODE REVIEW

File: practice-areas/project-management/accessibility-guidelines
Comment (dmundra) [inline practice-areas/project-management/accessibility-guidelines:1]: @kristenjernigan sorry if this is too early a review. The filename is missing the extension '.md'. After you make that change you see some additional prompts to add the new file to navigation and so on. Let me know if you would like any help with that.

**Review by dmundra — CHANGES_REQUESTED** [review]

File: practice-areas/project-management/accessibility-guidelines
Comment (dmundra) [reply practice-areas/project-management/accessibility-guidelines:1]: @kristenjernigan I fixed the extension of the file and made some minor updates.

**Review by grugnog — CHANGES_REQUESTED** [review]
I think it would be worth considering the purpose/location of this page - it seems like best accessibility practices for meetings should be common to all meetings, not just ones created by project managers?
I am also not sure we want captions always enabled, but rather that people can enable them when preferred or set them to always be on in their settings. Both of these are already always true for CivicActions Zoom meetings.
Camera to default on also has some accessibility nuance, probably better to link to this page instead: https://guidebook.civicactions.com/en/latest/company-policies/new-hire-orientation/video-call-best-practices/

**Comment by dmundra** [comment]
Double checked with @kristenjernigan that this is no longer needed.

**Pull request closed without merging**